NEWS.md

```diff
@@ -1,3 +1,33 @@
+## styler 1.0.1 (2018-03-02)
+
+This is a maintenance release without any breaking API changes.
+
+## Major & dependency related changes
+
+* Removed implicit `dplyr` dependency via `purrr:::map_dfr()` (thanks 
+  @jimhester, #324).
+* Added required minimal version dependency for purr (`>= 0.2.3`) (#338).
+* The dependency tibble was optimized for speed in `v1.4.2` so styler should run 
+  ~2x as fast [(#348)](https://github.com/tidyverse/tibble/pull/348). Hence, 
+  styler now depends on `tibble >= 1.4.2`.
+* In the dependency `enc`, a bug was fixed that removed/changed non-ASCII 
+  characters. Hence, styler now depends on `enc >= 0.1-10` (#348).
+
+## Minor changes
+
+* Recognizing and respecting of DSLs used in R comments: rplumnber (`#*`, #306), 
```

Comment:
"respecting of" doesn't sound right

Reply:
Really? I thought that is proper usage. Searching dplyr issues and PRs gives over 30 results that seem to use "respect" in the same way: https://github.com/tidyverse/dplyr/issues?utf8=✓&q=respect

Reply:
Maybe "We're now recognizing and respecting DSLs used in R comments. ...